pelican/settings.py

```diff
@@ -660,3 +661,16 @@ def configure_settings(settings):
             continue            # setting not specified, nothing to do
 
     return settings
+
+
+def coerce_overrides(overrides):
+    coerced = {}
+    setting_types = {k: type(v) for k, v in DEFAULT_CONFIG.items()}
```

Comment:
You don't need to construct this. You can look up as you go through overrides.

Reply:
Nixed that set and queried directly